src/fides/api/alembic/migrations/versions/fd52d5f08c17_system_dictionary_data_migration.py

```diff
@@ -0,0 +1,374 @@
+    """Migrating systems.joint_controller and datasets.joint_controller encrypted json string fields to
+    System.joint_controller_info which is just a string.
+
+    Prioritizing system joint_controller info if it exists, or taking the first non-null entry from datasets
+    otherwise.
+    """
+    new_joint_controller_info: Optional[str] = None
+    # Prioritize system data over dataset data
+    joint_controller_data = (
+        joint_controller_decrypted
+        or next(  # Multiple datasets can be related to a system, so we use the first non null one if applicable
+            (
+                item
+                for item in datasets_joint_controller_decrypted or []
+                if item is not None
+            ),
+            None,
+        )
+    )
+    if joint_controller_data and isinstance(joint_controller_data, dict):
+        separated_fields = [
+            joint_controller_data.get("name"),
+            joint_controller_data.get("address"),
+            joint_controller_data.get("email"),
+            joint_controller_data.get("phone"),
+        ]
+
+        new_joint_controller_info = "; ".join(
+            [elem for elem in separated_fields if elem and elem != "N/A"]
+        )
+        if not new_joint_controller_info:
+            # If this is an empty string, let's set it back to None
+            new_joint_controller_info = None
+
+    return new_joint_controller_info
+
+
+def _system_calculate_does_international_transfers(
+    system_third_country_transfers: Optional[List[str]],
+    dataset_third_country_transfers: Optional[List[str]],
+) -> bool:
+    """Migrating System.third_country_transfers and Dataset.third_country_transfers to System.does_international_transfers
+    If any countries are listed, regardless of which country it is, System.does_international_transfers will be set to true.
+    """
+    return bool(system_third_country_transfers) or bool(dataset_third_country_transfers)
+
+
+def _system_transform_data_responsibility_title_type(
+    dept_str: Optional[str],
+) -> List[str]:
+    """Migrating System.data_responsibility_title (str) to System.responsibility(list).  There can now be potentially many
+    items listed here"""
+    if not dept_str:
+        return []
+    return [dept_str]
+
+
+def _flatten_data_protection_impact_assessment(
+    data_protection_impact_assessment: Optional[Dict],
+) -> Tuple[bool, Optional[str], Optional[str]]:
+    """Flattening data from System.data_protection_impact_assessment into three separate fields:
+    System.requires_data_protection_assessments, System.dpa_location, and System.dpa_progress
+    """
+    if not data_protection_impact_assessment or not isinstance(
+        data_protection_impact_assessment, dict
+    ):
+        return False, None, None
+
+    requires_data_protection_assessments = data_protection_impact_assessment.get(
+        "is_required", False
+    )
+    dpa_progress = data_protection_impact_assessment.get("progress", None)
```

Comment:
Mega nit: `None` is the default fallback

```suggestion
    dpa_progress = data_protection_impact_assessment.get("progress")
```

Reply:
sure I can remove the default